webpack.config.js

```diff
@@ -30,7 +30,7 @@ module.exports = {
     
     // Omit websocket functionality from postgraphile:
     new webpack.NormalModuleReplacementPlugin(
-      /postgraphile\/build\/postgraphile\/http\/subscriptions\.js$/,
+      /postgraphile\/build-turbo\/postgraphile\/http\/subscriptions\.js$/,
       `${__dirname}/src/postgraphile-http-subscriptions.js`
     ),
```

Comment:
I don't think this is actually a fix unless you've set `GRAPHILE_TURBO` which I don't think is set by default.

Instead, please throw away the `yarn.lock` changes and duplicate this above block - once for non-turbo and once for turbo.

Thanks!